### PR TITLE
Feat/v0

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: Build/release
```
Comment (BotellaA) [inline .github/workflows/publish.yaml:1]: Maybe not yet in this PR? We should not publish for every push

File: electron-builder.json5 [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/**
```
Comment (BotellaA) [inline electron-builder.json5:1]: To remove

File: electron/main.js [hunk 3]
```diff
@@ -1,16 +1,41 @@
-const { app, BrowserWindow } = require("electron");
+import { app, BrowserWindow } from "electron";
+import { updateElectronApp } from "update-electron-app";
+import path from "path";
+
+updateElectronApp();
+
+process.env.ROOT = path.join(__dirname, "..", "..");
+process.env.DIST = path.join(process.env.ROOT, "dist-electron");
+process.env.VITE_PUBLIC = process.env.VITE_DEV_SERVER_URL
```
Comment (BotellaA) [inline electron/main.js:9]: Clean all unused variables

File: electron/preload.js [hunk 4]
```diff
@@ -0,0 +1 @@
+console.log("---- electron/preload.ts ----");
```
Comment (BotellaA) [inline electron/preload.js:1]: To remove?

File: forge.config.cjs [hunk 5]
```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  packagerConfig: {
+    asar: false,
```
Comment (BotellaA) [inline forge.config.cjs:3]: Does it work with asar true?

File: forge.config.cjs [hunk 6]
```diff
@@ -0,0 +1,26 @@
+module.exports = {
```
Comment (BotellaA) [inline forge.config.cjs:1]: Is it a js or cjs file? Js is enough I think

File: layouts/default.vue [hunk 7]
```diff
@@ -0,0 +1,39 @@
+<template>
```
Comment (BotellaA) [inline layouts/default.vue:1]: This file should be shown to @SpliiT so he does not have to also do it

File: nuxt.config.js [hunk 8]
```diff
@@ -7,13 +9,23 @@ export default defineNuxtConfig({
           : "localhost",
     },
   },
-  modules: ["nuxt-electron"],
+  modules: ["nuxt-electron", "vuetify-nuxt-module", VuetifyModule],
```
Comment (BotellaA) [inline nuxt.config.js:12]: You need both?

File: nuxt.config.js [hunk 9]
```diff
@@ -7,13 +9,23 @@ export default defineNuxtConfig({
...
   electron: {
     build: [
       {
+        // Main-Process entry file of the Electron App.
         entry: "electron/main.js",
       },
+      {
```
Comment (BotellaA) [inline nuxt.config.js:19]: To remove?

File: nuxt.config.js [hunk 10]
```diff
@@ -32,6 +44,9 @@ export default defineNuxtConfig({
     },
   },
 
+  build: {
+    transpile: ["vuetify"],
```
Comment (BotellaA) [inline nuxt.config.js:48]: Why do you need that?

File: nuxt.config.js [hunk 11]
```diff
@@ -32,6 +44,9 @@ export default defineNuxtConfig({
...
+  },
   vue: {
     compilerOptions: {
       isCustomElement: (tag) => ["md-linedivider"].includes(tag),
```
Comment (BotellaA) [inline nuxt.config.js:52]: What is that?

File: package.json [hunk 12]
```diff
@@ -1,12 +1,13 @@
 {
   "name": "vease",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "description": "Basic app",
-  "main": "electron/main.js",
+  "main": "dist-electron/main.js",
```
Comment (BotellaA) [inline package.json:5]: Dist electron should be committed or not?

File: nuxt.config.js (hunk 11 above)
Comment (JulienChampagnol) [reply nuxt.config.js:52]: Vue doesn't know when a component is local or from vuetify

**Review by BotellaA — APPROVED** [review]

File: package.json (hunk 12 above)
Comment (JulienChampagnol) [reply package.json:5]: In my opinion it shouldn't because it is generated, but this says to the build package where the entry point is

**Pull request merged**